src/WorkOS.net/WorkOS.net.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1</TargetFrameworks>
```

Comment:
Back in today from OOO, no we do not need to change the target framework, I will have to just tweak the time string comparison, since I used a function only available in 2.1+.

Reply:
Reverted Frameworks and removed FixedTimeString Equal function and added helper method to do it instead:
d33aea72c65739541a39cc12cb34c08c3fa90c20
bf00489fe5663c3422679218a033942b5ea385b8